templates/request/delete/delete.handlebars

```diff
@@ -19,8 +19,10 @@
           'Content-Type': '{{contentType}}'{{#if headerParameters}},
           'Custom-Header': {
           {{#each headerParameters}}
-          {{this.name}}: 'DATA GOES HERE'{{#unless @last}},{{/unless}}
-          {{/each}} }{{/if}}
+            {{this.name}}: 'DATA GOES HERE'{{#unless @last}},{{/unless}}
+          {{/each}} }{{/if}}{{#if basic}},
+          Authorization: 'Basic TOKEN GOES HERE'{{/if}}{{#if oauth}},
```

Comment:
I think we should use environment variables for secrets. People don't commit their secrets usually. As part of code generation we can have a `.env` file to hold the placeholders for the environment variables.

So here is should be something like: 

```js
Authorization: 'Basic ' + process.env.BASIC_AUTH_TOKEN
```